presto-verifier/src/test/java/com/facebook/presto/verifier/rewrite/TestQueryRewriter.java

```diff
@@ -568,6 +569,17 @@ public void testRewriteFunctionCalls()
                         "    AVG(a),\n" +
                         "    REPEAT(AVG(a), CAST(CARDINALITY(ARRAY[0.5, 0.9]) AS INTEGER))\n" +
                         "FROM test_table");
+
```

Comment:
This should be a separate and new test method that can pass or fail independently of the existing tests.